bims/templates/map.html

```diff
@@ -126,8 +127,8 @@
                     <div>MONTH</div>
                     <table id="month-selector" style="width:100%">
                     </table>
-                    <button id="apply-filter">Apply Filter</button>
-                    <button id="clear-filter">Clear Filter</button>
+                    <button class="apply-filter">Apply Filter</button>
+                    <button class="apply-filter">Clear Filter</button>
```

Comment:
please use class clear-filter, because we are using that class for event in here
https://github.com/ann26/django-bims/blob/d5c025cc36f8d85a4f5626d7377e8c87ac1e9e57/bims/static/js/views/search.js#L12